packages/codegen/src/generateEntityCodegenFile.ts

```diff
@@ -924,19 +915,17 @@ function createPgEnums(meta: EntityDbMetadata) {
   });
 }
 
-function createRelations(config: Config, meta: EntityDbMetadata, entity: Entity) {
+function createRelations(config: Config, meta: EntityDbMetadata, entity: Entity): Relation[] {
   // Add ManyToOne entities
   const m2o: Relation[] = meta.manyToOnes.map((m2o) => {
     const { fieldName, otherEntity, otherFieldName, notNull } = m2o;
     const maybeOptional = notNull ? "never" : "undefined";
     if (m2o.derived === "async") {
-      const line = code`
-        abstract readonly ${fieldName}: ${ReactiveReference}<${entity.name}, ${otherEntity.type}, ${maybeOptional}>;
-      `;
+      const line = code`abstract readonly ${fieldName}: ${ReactiveReference}<${entity.name}, ${otherEntity.type}, ${maybeOptional}>;`;
       return { kind: "abstract", line } as const;
     }
     const decl = code`${ManyToOneReference}<${entity.type}, ${otherEntity.type}, ${maybeOptional}>`;
-    const init = code`${hasOne}(this, ${otherEntity.metaType}, "${fieldName}", "${otherFieldName}")`;
+    const init = code`${hasOne}("${otherFieldName}")`;
```

Comment:
Updated all of the `has...` methods to not need `this` or `fieldName` or `otherMetadata`